django-cloudlaunch/baselaunch/views.py

```diff
@@ -531,21 +531,38 @@ class DeploymentViewSet(viewsets.ModelViewSet):
     """
     List compute related urls.
     """
-    queryset = models.ApplicationDeployment.objects.all()
+    # queryset = models.ApplicationDeployment.objects.all()
```

Comment:
Just removed it. It get overwritten by the `get_queryset` method below (https://stackoverflow.com/a/29015663)